### DISCUSSION TITLE
Percentage symbol before the number

### DISCUSSION THREAD

**Discussion by xabirequejo** [elk-zone/elk#2846] [Q&A] [answered]
Hi,

Is there any way to place the percentage symbol before the number?
This is the string that I'd like to tweak:
`"status": "Translation status: {0}/{1} ({2}%)",`

But my solution doesn't work (`"status": "Itzulpenaren egoera: {1} kateetatik {0} itzulita (%{2}a)",`), it's not showing the symbol at all.
I see that some languages use the ٪ symbol. Is this the best solution? Is there any other workaround?

Thanks

**Comment by shuuji3** [comment, marked as answer]
Hi @xabirequejo This is because `%{...}` is treated as a special syntax by the localization library vue-i18n (ref. Formatting | Vue I18n - https://kazupon.github.io/vue-i18n/guide/formatting.html#support-ruby-on-rails-i18n-format). Please try escaping the character with `{'%'}`:

```
      "status": "Translation status: {0}/{1} ({'%'}{2})",
```

**Reply by xabirequejo** [reply]
Will do. Thank you very much.

**Reply by shuuji3** [reply]
You're welcome! It looks working as expected. 🙂
